### PR TITLE
Migrate to Deno

### PR DESCRIPTION
**Pull request by toshmukhamedov**

### CODE REVIEW

File: src/cron/cron.ts [hunk 1]
```diff
@@ -153,8 +152,10 @@ async function daily(bot: Bot<BotContext>) {
 }
 
 async function reminder(bot: Bot<BotContext>) {
-  await schedule.gracefulShutdown();
```
Comment (Xayrulloh) [inline src/cron/cron.ts:156]: what about this one

File: src/server.ts [hunk 2]
```diff
@@ -29,7 +28,7 @@ bot.api.config.use(
 // middleware
 bot.use(
   session({
-    initial: () => (Promise.resolve({} as any)),
+    initial: () => ({}),
```
Comment (Xayrulloh) [inline src/server.ts:31]: what it does?

File: src/server.ts [hunk 3]
```diff
@@ -140,13 +139,24 @@ void cronStarter(bot);
...
 // webhook
 if (env.WEBHOOK_ENABLED) {
-  const server = Fastify();
-
-  server.post(`/${env.TOKEN}`, webhookCallback(bot, "fastify"));
-  server.setErrorHandler(errorHandler);
-  server.listen({ port: env.WEBHOOK_PORT }, async () => {
-    await bot.api.setWebhook(env.WEBHOOK_URL + env.TOKEN);
-  });
+  env.WEBHOOK_URL = "https://3ac4-84-54-66-241.ngrok-free.app/";
```
Comment (Xayrulloh) [inline src/server.ts:142]: wait a minute, why you didn't take it from .env 🤔

File: src/server.ts [hunk 4]
```diff
@@ -140,13 +139,24 @@ void cronStarter(bot);
...
+  env.WEBHOOK_PORT = 8000;
```
Comment (Xayrulloh) [inline src/server.ts:143]: ditto

File: src/server.ts [hunk 5]
```diff
@@ -140,13 +139,24 @@ void cronStarter(bot);
...
+
+  const secretToken = env.TOKEN.split(":")[1];
```
Comment (Xayrulloh) [inline src/server.ts:145]: can you explain what's going on in here pls

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 4 above)
Comment (toshmukhamedov) [reply src/server.ts]: fixed

File: src/server.ts (hunk 5 above)
Comment (toshmukhamedov) [reply src/server.ts:142]: we use token as auth

File: src/server.ts (hunk 2 above)
Comment (toshmukhamedov) [reply src/server.ts:31]: initialize data for session

Comment (Xayrulloh) [reply src/server.ts:31]: yeah i just wonder what caused you to change it, is it gave any errors? or smth like this or it's better approach or etc..

File: src/server.ts (hunk 4 above)
Comment (Xayrulloh) [reply src/server.ts]: btw ditto means "same" but in japan))

File: src/server.ts (hunk 5 above)
Comment (Xayrulloh) [reply src/server.ts:142]: ok

File: src/cron/cron.ts (hunk 1 above)
Comment (toshmukhamedov) [reply src/cron/cron.ts:156]: we don’t use node-schedule anymore

File: src/server.ts (hunk 2 above)
Comment (toshmukhamedov) [reply src/server.ts:31]: oldin o’zi shunaqa turudi

File: src/cron/cron.ts (hunk 1 above)
Comment (Xayrulloh) [reply src/cron/cron.ts:156]: I mean it's used to delete all memorized crones right so do we should use smth like this or deno cron itself forgets

File: src/server.ts (hunk 2 above)
Comment (Xayrulloh) [reply src/server.ts:31]: aaa ok

File: src/cron/cron.ts (hunk 1 above)
Comment (toshmukhamedov) [reply src/cron/cron.ts:156]: it’ll be cleaned by gc

Comment (Xayrulloh) [reply src/cron/cron.ts:156]: sorry but what is "gc"

Comment (toshmukhamedov) [reply src/cron/cron.ts:156]: `croner` doesn’t have such a functionality

Comment (toshmukhamedov) [reply src/cron/cron.ts:156]: garbage collector

**Pull request merged**